def get_exception(request):
    raise Exception('Demo exception')
